curriculum/challenges/english/25-front-end-development/quiz-css-layout-and-effects/66ed8ff4f45ce3ece4053eb4.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
+To automatically fix broken layout issues
 
 #### --answer--
 
-Placeholder answer
+To prevent data loss and make issues noticeable
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you handle overflow in CSS to hide content or add scrollbars?
 
 #### --distractors--
 
-Placeholder distractor 1
+Use `overflow: visible` to hide content and `overflow: auto` to always display scrollbars
 
 ---
 
-Placeholder distractor 2
+Use `overflow: collapse` to hide content and `overflow: expand` to display scrollbars
 
 ---
 
-Placeholder distractor 3
+Use `overflow: none` to hide content and `overflow: show` to display scrollbars
 
 #### --answer--
 
-Placeholder answer
+Use `overflow: hidden` to hide content and `overflow: scroll` to always display scrollbars
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+How can you specify different scrolling behaviors for the x-axis and y-axis in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+By using `scroll-x` and `scroll-y` properties
 
 ---
 
-Placeholder distractor 2
+By using `overflow-x` and `overflow-y` separately in every case
 
 ---
 
-Placeholder distractor 3
+By passing two values to the `overflow` property, where the first applies to `overflow-y` and the second to `overflow-x`
 
 #### --answer--
 
-Placeholder answer
+By passing two values to the `overflow` property, where the first applies to `overflow-x` and the second to `overflow-y`
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `transform` CSS property allow you to do?
 
 #### --distractors--
 
-Placeholder distractor 1
+Change the color and font style of an element
 
 ---
 
-Placeholder distractor 2
+Adjust the margin and padding of an element
 
 ---
 
-Placeholder distractor 3
+Modify the visibility and display of an element
 
 #### --answer--
 
-Placeholder answer
+Rotate, scale, skew, or translate an element
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `transform-box: fill-box` property do in an animation that rotates an element?
 
 #### --distractors--
 
-Placeholder distractor 1
+It resizes the element to fit the canvas during the transformation
 
 ---
 
-Placeholder distractor 2
+It changes the background of the element during transformation
 
 ---
 
-Placeholder distractor 3
+It sets the transform origin to the top-left corner of the element
 
 #### --answer--
 
-Placeholder answer
+It sets the transform origin to the center of the element's bounding box, allowing it to spin in place 
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What does the `transform-origin` property do in CSS?
 
 #### --distractors--
 
-Placeholder distractor 1
+It defines the size of transformed elements
 
 ---
 
-Placeholder distractor 2
+It alters the opacity of transformed elements
 
 ---
 
-Placeholder distractor 3
+It adjusts the color of transformed elements
 
 #### --answer--
 
-Placeholder answer
+It changes the position of transformed elements
```

Comment:
It's not technically true. While it changes the position in which the transformations take place, it's ultimate position is not changed.